src/index.html

```diff
@@ -1,14 +1,63 @@
 <!doctype html>
-<html lang="en">
+<html class="html" lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <div class="container">
+        <div class="img-conteiner">
+          <a class="img-link" href="#home">
+            <img
+              class="img"
+              src="images/logo.png"
+              alt="logo"
+            >
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav-list" >
+            <li>
+              <a class="link is-active" href="/">Apple</a>
+            </li>
+            <li>
+              <a class="link" href="#samsung">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li>
              <a class="link is-active" href="/">Apple</a>
            </li>

            <li>
              <a class="link" href="#samsung">Samsung</a>
            </li>
```

Add blank lines between multiple sibling blocks